### PR TITLE
New package: megapixels-1.3.0

### PR DESCRIPTION
**Pull request by jcgruenhage**
#### General
- [x] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [ ] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

**Comment by jcgruenhage** [comment]
Not sure, but there's a reason this is still marked WIP, I haven't gotten to building and trying it out lately.

**Comment by jcgruenhage** [comment]
I've just tested this on my Pinephone and gosh damn, this is so much better than it used to be. Works just fine.

**Comment by Piraty** [comment]
did it require a config file ?

**Comment by jcgruenhage** [comment]
No

**Comment by jcgruenhage** [comment]
So, regarding whether it needed a config file: megapixels itself ships a few config files out of the box which ensure that support for the pinephone and pinetab just works out of the box:

```
/usr/share/megapixels/config/pine64,pinephone-1.0.ini
/usr/share/megapixels/config/pine64,pinephone-1.1.ini
/usr/share/megapixels/config/pine64,pinephone-1.2.ini
/usr/share/megapixels/config/pine64,pinetab.ini
```

Thanks for the pointer in IRC that there's a newer release, package is updated, new package again tested on the pinephone and it all works as expected.

I'll need to figure out what to do with these .dng files that megapixels produces, but I'll surely find a way.

File: srcpkgs/megapixels/template [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+version=1.4.3
+revision=1
+build_style=meson
+hostmakedepends="pkg-config glib-devel"
+makedepends="gtk4-devel libzbar-devel"
+short_desc="GTK4 camera app for ARM devices"
+maintainer="Jan Christian Grünhage <jan.christian@gruenhage.xyz>"
+license="GPL-3.0-or-later"
+homepage="https://gitlab.com/postmarketOS/megapixels"
+distfiles="${homepage}/-/archive/${version}/megapixels-${version}.tar.gz"
```
Comment (paper42) [inline srcpkgs/megapixels/template:12]:
```suggestion
distfiles="https://gitlab.com/postmarketOS/megapixels/-/archive/${version}/megapixels-${version}.tar.gz"
```

Comment (jcgruenhage) [reply srcpkgs/megapixels/template:12]: Why? This makes the line longer and the url is already full of variables anyway.

Comment (Piraty) [reply srcpkgs/megapixels/template:12]: convention is to avoid it for distfiles ( to have better grep support i think)

**Pull request closed without merging**